### ISSUE TITLE
Update API version compliance

### ISSUE THREAD

**Issue by RichJeanes** [studioimaginaire/phue#80]
Any chance to see phue updated with support for the latest API version?
Appears to be 1.15 according to http://www.developers.meethue.com/documentation/changelog

I noticed the API version was an issue as the 'time' argument on the Schedule interface is deprecated in favor of 'localtime' (a change that seems to be missing from their changelog... So maybe it isn't all inclusive... How useful.) as of API 1.2.1 (in addition to 'status' and 'autodelete' being added in 1.2.1 and 1.3, respectively). However this means that I cannot create schedules at all using phue, just getting an error when I try.

I am happy to contribute, though I can't make any promises on quality or timeframe. 👍

**Comment by natcl** [comment]
Sure I'd be happy for a pull request !

**Comment by RichJeanes** [comment]
I'm going to start working on an overhaul over on my fork. I've been looking for a side project lately, anyway. Like I say, no promises on time frame, but it should be better than nothing. Mostly just updating the individual interfaces listed on the API documentation.

Thanks for putting this module together to begin with, though. I found it looking for a way to schedule my lights to turn on at sunset and I've managed it using phue, cron, and PyEphem.

(Though I have to ask why it isn't PyHue. Doesn't help that there's a Phue (PHP) that initially pushed to GitHub just nine days after you did.)
